misk/src/main/kotlin/misk/web/actions/NotFoundAction.kt

```diff
@@ -2,18 +2,24 @@ package misk.web.actions
 
 import misk.web.Get
 import misk.web.PathParam
+import misk.web.Post
+import misk.web.RequestContentType
 import misk.web.Response
 import misk.web.ResponseContentType
 import misk.web.mediatype.MediaTypes
+import okhttp3.Headers
 import javax.inject.Singleton
 
 @Singleton
 class NotFoundAction : WebAction {
     @Get("/{path:.*}")
-    @ResponseContentType(MediaTypes.TEXT_PLAIN_UTF8)
+    @Post("/{path:.*}")
+    @RequestContentType(MediaTypes.ALL)
+    @ResponseContentType(MediaTypes.ALL)
```

Comment:
why is the response type not plain text?

Reply:
I'm guessing this is required so it's not dependent on what the request accepts (e.g if the request accepts json, this should still work)

Reply:
correct